File: generic/codes/googleJam/gcj/gcj2012/Quali/c/Quali_2012_c.py
t = raw_input()
t = int(t)
for i in range(1, t + 1):
    line = raw_input()
    a, b = line.split()
    a = int(a)
    b = int(b)
    res = 0
    for j in range(a, b + 1):
        jStr = str(j)
        l = len(jStr)
        jStr += jStr
        for k in range(l):
            num = int(jStr[k:l+k])
            if(num >= a and num <= b):
                if(num != j):
                    res += 1
                    print str(num) + " : " + jStr
    print "Case #" + str(i) + ": " + str(res / 2)

File: generic/codes/googleJam/gcj/gcj2012/Quali/c/Quali_2012_c.c
#include <sdtio.h>

int main()
{
	int t, i, a, b, j, res, tj, k;
	int al[14];

	scanf("%d", &t);
	for (i = 1; i <= t; ++i)
	{
		res = 0;
		scanf("%d %d", &a, &b);
		for(j = a; j <= b; ++j)
		{
			k = 13;
			tj = j;
			while(tj)
			{
				al[k--] = a % 10;
				a /= 10;
			}
			
		}
	}

	return 0;
}
